### PR TITLE
Badges and input

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Please check the review and review the code thanks alott

### CODE REVIEW

**Comment by partha8** [comment]
Hey @Dev-AkshatGupta, I cant access badges and input, you didn't link it.

**Comment by Dev-AkshatGupta** [comment]
> Hey @Dev-AkshatGupta, I cant access badges and input, you didn't link it.
Yeah gonna solve this issue

File: html-folder/badges.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                <p class="text-black">Badges are small circles, positioned either at top-right or bottom-right of the parent component. Badge can be used to display numbers, online / offline status, depending on where they are used.
+                </p>
+                <div class="divider margin-bot-lrg margin-top-lrg"></div>
+                <h2>Profile Badges</h2>
+                <p class="text-black">
+                    Profile Badge is used to show the current status of a user.
+                </p>
+                <div class="components-representation-block flex-space-evenly-center smooth-square-radius">
+                    <div class="avatar avatar-md relative code-repre-padd">
+                        <img src="../assets/pexels-cottonbro-6954244.jpg" alt="">
```
Comment (ParthParolekar) [inline html-folder/badges.html]: Add `alt` to images

Comment (Dev-AkshatGupta) [reply html-folder/badges.html]: Done can you please approve it

**Review by partha8 — APPROVED** [review]
great work!

**Pull request merged**